semconv/v1.4.0/schema.go

```diff
@@ -17,4 +17,4 @@ package semconv // import "go.opentelemetry.io/otel/semconv/v1.4.0"
 // SchemaURL is the schema URL that matches the version of the semantic conventions
 // that this package defines. Semconv packages starting from v1.4.0 must declare
 // non-empty schema URL in the form https://opentelemetry.io/schemas/<version>
-const SchemaURL = "https://opentelemetry.io/schemas/v1.4.0"
+const SchemaURL = "https://opentelemetry.io/schemas/1.4.0"
```

Comment:
Just to confirm: this file is not auto-generated, right?

Reply:
Correct, currently it is not. :+1: